### ISSUE TITLE
Show error for invalid keybindings

### ISSUE THREAD

**Issue by waderyan** [Microsoft/vscode#14478]
I opened https://github.com/Microsoft/vscode/issues/14323#issuecomment-256072917. There are some keybindings that other products support that we do not support. 

This [doc](https://code.visualstudio.com/docs/customization/keybindings#_accepted-keys) outlines the keys that are accepted. 

Keys that are missing that I have found while making Atom and Sublime Text keymap extensions (I'd imagine there will be others) are: `context_menu`, `}`, `{`, and `+`. 

We should consider adding support for these.

**Comment by alexandrudima** [comment]
@waderyan `{`, `}` and `+` are not keys that we are lacking. Sorry, but please read the documentation again or perhaps the documentation is incomplete or poorly written? Please submit a PR to improve it in this case (English is not my mother tongue and the topic is a bit complicated to begin with)

https://code.visualstudio.com/docs/customization/keybindings#_keyboard-layouts

![image](https://cloud.githubusercontent.com/assets/5047891/19727333/3fe8c040-9b8f-11e6-9614-308821a7cf59.png)

I agree that we are lacking `context_menu`.

**Comment by waderyan** [comment]
@alexandrudima thank you for your response and patience with me as I try and figure this out. I see what I am missing now. I will take the look at the docs and make a PR if I think it can be improved. 

(What I was missing was that `{` key is invalid because how would it be invoked when `shift`+`[` is valid?)

**Comment by alexandrudima** [comment]
Yeah, and to make matters more complicated `[` is just a symbolic representation for `VK_OEM_4` which under different keyboard layouts will not produce `[` when typed. So the best way to think about them is as key codes. `[` is the symbol for talking about `VK_OEM_4` because it is hard to talk about `VK_OEM_4`. It is not a coincidence, pressing `VK_OEM_4` in the US kb layout produces `[` but that is not the case in other layouts.

For example:
* `` ` `` (`VK_OEM_3`) produces a combining circumflex accent (`^`) on German kb layouts -- https://github.com/Microsoft/vscode/issues/12682#issuecomment-256000312

**Comment by engelfrost** [comment]
I've found that I cannot use the `§` key on my Swedish keyboard for key bindings, from what I understand this is related to this issue. I'm not sure how to find the key code.
